1-js/04-object-basics/01-object/article.md

```diff
@@ -7,7 +7,7 @@ In contrast, objects are used to store keyed collections of various data and mor
 
 An object can be created with figure brackets `{…}` with an optional list of *properties*. A property is a "key: value" pair, where `key` is a string (also called a "property name"), and `value` can be anything.
 
-We can imagine an object as a cabinet with signed files. Every piece of data is stored in its file by the key. It's easy to find a file by its name or add/remove a file.
+We can imagine an object as a cabinet with labeled folders. Every piece of data is stored in its folder by the key. It's easy to find a folder by its name or add/remove a folder.
```

Comment:
file is clear IMO, see [File cabinet wiki](https://en.wikipedia.org/wiki/Filing_cabinet)